package.json

```diff
@@ -236,6 +236,7 @@
   },
   "dependencies": {
     "mitt": "3.0.1",
+    "selenium-webdriver": "^4.15.0",
```

Comment:
Since this is not a run-time dependency, let’s move it to `devDependencies`?

Reply:
Also the dependency needs to be pinned.

Reply:
done

Reply:
done